client/components/emails/email-step-side-bar/index.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Icon } from '@wordpress/icons';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import Gridicon from "calypso/components/gridicon";
+
+class EmailStepSideBar extends React.Component {
+	getExplainers() {
+		const { translate } = this.props;
+
+		const title = translate(
+			'{{b}}75%{{/b}} of U.S. e-commerce customers prefer to trust a business with a custom email',
+			{
+				components: { b: <strong /> },
+			}
+		);
+		const subtitle = (
+			<>
+				<p>
+					<Gridicon icon="checkmark" size={ 16 } />
+					{ `${ translate( 'Stand out with every email you send' ) }` }
+				</p>
+				<p>
+					<Gridicon icon="checkmark" size={ 16 } />
+					{ `${ translate( 'Single dashboard for your email, domain and website' ) }` }
+				</p>
+				<p>
+					<Gridicon icon="checkmark" size={ 16 } />
+					{ `${ translate( 'Easy to import your existing emails and contacts' ) }` }
```

Comment:
I don't think we need the wrapped/nested construction here - `translate` returns a `TranslateResult` type (which you can look up) -- it's either a string or a valid React fragment, so can be embedded directly and safely.
```
{ `${ translate( ... ) }` } -> { translate( ... ) }
```